python/hsfs/core/statistics_api.py

```diff
@@ -66,7 +66,7 @@ def get(
         end_commit_time: Optional[int] = None,
         feature_names: List[str] = None,
         row_percentage: Optional[float] = None,
-        before_transformation: bool = False,
+        before_transformation: Optional[bool] = None,
```

Comment:
It is a set it back to None because it was breaking things. Basically setting to True or False make it into a query parameters which not accepted for FeatureGroups, so setting it to None is necessary for now